jembi/jempi-web
# feat: add keycloak sso

## Motivation
This PR adds support for signing in using KeyCloak.

## How to test

1. Rename the following KeyCloak Client config to ".json" and import it into KeyCloack :
[jempi-oauth.json.txt](https://github.com/jembi/jempi-web/files/10371352/jempi-oauth.json.txt)
2. Access JeMPI "/login" and click on "Sign In with KeyCloak"
3. Authenticate to KeyCloack
4. Should redirect back to JeMPI
